authors/apps/authentication/token.py

```diff
@@ -0,0 +1,16 @@
+import jwt
+import datetime
+
+from django.conf import settings
+
+
+def generate_jwt_token(email, username):
+    """Method to generate user jwt token."""
+    time = datetime.datetime.utcnow() + datetime.timedelta(seconds=86400)
+    token = jwt.encode({
+        "username": username,
+        "email": email,
```

Comment:
I notice you are using both email and username as payload. I would recommend we use just the username for security and privacy concerns.

Reply:
Thank you for the insight. Let us rectify that and use the username since it is also unique for all users. We are working on that.